test/airports_test.exs

```diff
@@ -2,6 +2,6 @@ defmodule AirportsTest do
   use ExUnit.Case
 
   test "all/0 returns all the airports" do
-    assert Enum.count(Airports.all()) == 7184
+    assert Enum.count(Airports.all()) == 69197
```

Comment:
@christianlj @samhamilton I would suggest asserting the number of rows of airports from the csv should be equal to `Enum.count(Airports.all())`. And that all airports have non-null fields (if it applies).